pkg/packagekit/kpatch.jsx

```diff
@@ -150,7 +150,7 @@ export class KpatchSettings extends React.Component {
                                 })
                             );
                 })
-                .catch(console.error);
+                .catch(err => console.error("Could not determine kernel version:", JSON.stringify(err)));
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19511-20231020-065404-313488a1-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)